house/people.go

```diff
@@ -3,54 +3,77 @@ package house
 import (
 	"context"
 	"fmt"
-	"github.com/google/uuid"
-	"github.com/prometheus/client_golang/prometheus"
-	"github.com/prometheus/client_golang/prometheus/promauto"
+	pb "github.com/beaujr/nmap_prometheus/proto"
+	etcdv3 "github.com/ozonru/etcd/v3/clientv3"
 	"gopkg.in/yaml.v2"
 	"log"
-	"time"
 )
 
-type people struct {
-	Id      string   `json:"id",yaml:"id"`
-	Devices []string `json:"devices",yaml:"devices"`
-	Name    string   `json:"name",yaml:"name"`
+var peoplePrefix = "/people/"
+
+// Notifier represents and interface for notification sending
+type PeopleManager interface {
+	Get() ([]*pb.Person, error)
+	CreateFromDevices(devices []*pb.Devices, name string) error
+	Create(ids []string, name string) error
+}
+
+// NewNotifier returns a new Notifier
+func NewPeopleManager(etcdClient etcdv3.KV) PeopleManager {
+	return &EtcdPeopleManager{etcdClient: etcdClient}
 }
 
-func (s *Server) createPerson(devices []string, name string) error {
-	// Create a queue
-	person := &people{
-		Name:    name,
-		Devices: devices,
-		Id:      uuid.New().String(),
+// FCMNotifier is an implementation of the Notifier
+type EtcdPeopleManager struct {
+	Notifier
+	etcdClient etcdv3.KV
+}
+
+func (etm *EtcdPeopleManager) Get() ([]*pb.Person, error) {
```

Comment:
warning: exported method EtcdPeopleManager.Get should have comment or be unexported (golint)